alembic/versions/205ce02374b3_add_received_collateral_address_to_liquidations.py

```diff
@@ -0,0 +1,27 @@
+"""Add received_collateral_address to liquidations
+
+Revision ID: 205ce02374b3
+Revises: c8363617aa07
+Create Date: 2021-10-04 19:52:40.017084
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "205ce02374b3"
+down_revision = "c8363617aa07"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "liquidations",
+        sa.Column("received_token_address", sa.String(256), nullable=True),
```

Comment:
what happens if i call inspect block after merging the code and before applying the migration?

since the field is in the model, i'm wondering if it'll say `received_token_address` doesn't exist on save or if it'll just ignore the field on the model

Reply:
Have you tried it? I think SQLAlchemy will attempt to write to it

Can test by downgrading to remove the column